### ISSUE TITLE
Create overall design

### ISSUE THREAD

**Issue by lucamaraschi** [microbial-lab/contrib#3]
I think we should sketch what we are trying to achieve as I feel we might end up with plenty of repos and forget how they are connected together to achieve a common goal.

**Comment by pelger** [comment]
Agreed - I'm short on bandwidth this week but happy to take a first cut next week
